aspnetcore/security/authentication/passkeys-migration.md

```diff
@@ -0,0 +1,334 @@
+---
+title: Implement passkeys in an existing ASP.NET Core Razor Pages or MVC app
```

Comment:
> What we can do is leave the (half-baked) article in place and just remove it from the ToC for now. This all assumes that you're planning passkey support for RP/MVC.

Passkeys are supported in RP/MVC, as the Identity passkey APIs don't depend on a particular rendering technology, but we don't currently plan to update the Razor Pages and MVC templates to utilize passkeys. So I'd be inclined to hold off on this particular guide for now.